done/static/js/src/done.js

```diff
@@ -1,33 +1,51 @@
-function DoneXBlockOn(runtime, element) {
-    $('.done_windshield', element).addClass("done_windshield_off").removeClass("done_windshield_on");
-    DoneXBlock(runtime, element);
+/* Dummy code to make sure events work in Workbench as well as
+ * edx-platform*/
+if (typeof Logger === 'undefined') {
+    var Logger = {
+        log: function(a, b) { return; }
+    }
 }
 
-function DoneXBlockOff(runtime, element) {
-    console.log("b");
-    $('.done_windshield', element).addClass("done_windshield_on").removeClass("done_windshield_off");
-    DoneXBlock(runtime, element);
+function update_knob(element, data) {
+  if($('.done_onoffswitch-checkbox', element).prop("checked")) {
```

Comment:
These changes should be done via classes in the CSS-- it is very unusual to directly change the CSS via JavaScript.